apis/swagger.yml

```diff
@@ -2211,6 +2211,13 @@ definitions:
         description: "net priority."
         type: "integer"
         default: 0
+      SpecificID:
+        type: "string"
+        description: |
+          Create container with given id.
+          The length of given id should be 64.
+          The characters of given id should be in 0123456789abcdef.
+          By default, given id is unnecessary.
```

Comment:
Could you add the length here to help illustrate this?
```
minLength: 64
maxLength: 64
```

Reply:
Have fixed it!